playground/app/src/main/java/com/alibaba/weex/SimulatorDebugActivity.java

```diff
@@ -0,0 +1,100 @@
+package com.alibaba.weex;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+import com.taobao.weex.WXEnvironment;
+import com.taobao.weex.WXSDKEngine;
+
+import java.util.Locale;
+
+/**
+ * 模拟器debug页面
+ * @author Senyu email: senyu.ylj@alibaba-inc.com
+ */
+public class SimulatorDebugActivity extends AppCompatActivity {
+    private ImageView mBackImageView;
+    private EditText mDebugHostEditText;
+    private EditText mDebugPortEditText;
+    private EditText mServerEditText;
+    private Button mConnectDevToolButton;
+    private Button mConnectServerButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_simulator);
+        initView();
+        initListener();
+    }
+
+    /**
+     * 初始化视图
+     */
+    private void initView(){
+        mBackImageView = (ImageView) findViewById(R.id.simulator_back);
+        mDebugHostEditText = (EditText) findViewById(R.id.simulator_debug_host);
+        mDebugPortEditText = (EditText) findViewById(R.id.simulator_debug_port);
+        mServerEditText = (EditText) findViewById(R.id.simulator_server);
+        mConnectDevToolButton = (Button)findViewById(R.id.simulator_connect_devtool);
+        mConnectServerButton = (Button)findViewById(R.id.simulator_connect_server);
+    }
+
+    /**
+     * 初始化监听
+     */
+    private void initListener(){
+        mConnectDevToolButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if(!mDebugHostEditText.getText().toString().
+                        matches("\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}")){
+                    Toast.makeText(SimulatorDebugActivity.this, "请输入有效服务器Ip", Toast.LENGTH_SHORT).show();
```

Comment:
String constant write in values.xml would be better, that is convenient to add descriptor of multi-language.